rest-api/src/main/resources/paths/index.yml

```diff
@@ -613,6 +613,10 @@
     $ref: ./studies/v5_studies_studyId_adherence_weekly.yml
 /v5/studies/{studyId}/adherence/stats:
     $ref: ./studies/v5_studies_studyId_adherence_stats.yml
+/v5/studies/{studyId}/alerts:
+    $ref: ./studies/v5_studies_studyId_alerts.yml
+/v5/studies/{studyId}/deleteAlerts:
```

Comment:
As per previous conversation, we should change this URL to `/v5/studies/{studyId}/alerts/delete`.